rooms.js

```diff
@@ -306,6 +307,22 @@ let Room = (() => {
 		}
 		return successUserid;
 	};
+	Room.prototype.writeModlog = function (text) {
+		if (!this.modlog) return;
+		this.modlog.write('[' + (new Date().toJSON()) + '] ' + text + '\n');
+	};
+	Room.prototype.sendModCommand = function (data) {
+		for (let i in this.users) {
+			let user = this.users[i];
+			// hardcoded for performance reasons (this is an inner loop)
+			if (user.isStaff || (this.auth && (this.auth[user.userid] || '+') !== '+')) {
+				user.sendTo(this, data);
+			}
+		}
+	};
+	Room.prototype.modLog = function (text) {
```

Comment:
Also having two different functions `modlog` and `writemodlog` is kind of confusing - do we really need both?